docs/debugging-travis.md

```diff
@@ -0,0 +1,47 @@
+### Debugging Travis Build
+
+Kritis uses Travis to run CI unit tests [here](https://travis-ci.org/grafeas/kritis).
+
+You can use `Docker` to create similar container environment for local testing.
+
+This is helpful if you cannot reproduce Travis failures in your local dev environment.
+
+
+1. Find the latest `xenial` image tag at the travis dockerhub [repo](https://hub.docker.com/r/travisci/ci-sardonyx/tags). 
+As an example, the latest image tag as of writing is `packer-1564753982-0c06deb6`.
+
+```shell
+$ IMAGE="travisci/ci-sardonyx:packer-1564753982-0c06deb6"
+```
+2. Start a Docker container.
+
+```shell
+$ BUILDID="build-$RANDOM"
```

Comment:
How's the `RANDOM` picked here? just manually by the user?

Reply:
This reuses the `RANDOM` system environment variable.

Reply:
Oh, gotcha!